### ISSUE TITLE
Increased throttling starting 2021-09-23?

### ISSUE THREAD

**Issue by danlinenberg** [ranaroussi/yfinance#841]
This probably isn't a yfinance issue, but I didn't find any other place to ask this -

I noticed that starting 2021-09-23, after querying historical prices for a few tickers, Yahoo suddenly stops providing data for the last 2 weeks. 
When going straight into Yahoo's UI (https://finance.yahoo.com/quote/AAPL/history) I can see starting from 2021-09-07 - there is no price data at all.
When I reset my IP or wait long enough - the problem gets resolved, so this seems intentional.

Does anyone else experience something similar to this?

**Comment by kumarunster** [comment]
@danlinenberg yes, same here, trying to spread the work across different ips.

**Comment by patric2021** [comment]
@danlinenberg same here. I added a sleep 2-5 seconds between the tickers, and it seemed to work for now. But I cannot be sure if all data are correct and up to date. Don't know if it's throttling, or hitting a cache server with outdated data. Really annoying as it's unpredictable. Anyone know a better way of solving this issue? 

btw, is yfinance version related to this problem? I'm using the latest 0.1.63. Thanks.

**Comment by rahimkhoja** [comment]
I have this issue.

**Comment by heduobacht** [comment]
I got a similar issue.
But it does not look to be throtteling and also the mentioned 23rd September seems to be random.
These dates can be fetched on nearly every stock.
2021-08-31
2021-09-01
2021-09-02
2021-09-03
2021-09-30
2021-10-01
2021-10-05

I suspect it has something to do with (Multi-)Threading of the download. By turning off the multithreading it is far slower but it seems to get all the data from yahoo until today correctly. 
Tried it with Daily data for 1 Year of 13 Stocks so unlikely that Throtteling hits the limit with this small amount of data.

Many other issues hint to the same problem in September, but i have no clue where to search for a solution
